pom.xml

```diff
@@ -26,7 +26,7 @@
   <groupId>com.baloise.open.corellia</groupId>
```

Comment:
Having the same group and artifact id is considered a bad practice

Reply:
accepted